Monopoly/Assets/Scripts/Player.cs

```diff
@@ -16,14 +16,24 @@ public class Player : MonoBehaviour
 
     int currentFieldId;
 
+
     public List<Property> ownedProperties;
 
+
     public void MoveToPosition(int index) //przesunięcie na wybraną pozycję
     {
         currentFieldId = index;
         if (!pawn.IsDestinationReached())
             pawn.AllowMovement(index);
     }
+    public void MoveByNumberOfFields(int number)
+    {
+        currentFieldId += number;
```

Comment:
chyba zła nazwa zmiennej, currentFieldIndex, bo Id nie powinno się zmieniać (wiem, że to niekoniecznie Ty zrobiłeś, bo było tak już wcześniej :P ale można przy okazji poprawić)

Reply:
jedyny błąd to taki, że ja nieodpowiednio nazwałem zmienną
bo oznacza ona oczywiście index